mqtt/client/mqsub.sh

```diff
@@ -14,7 +14,9 @@ fi
 echo 0 > /proc/led/freq
 
 log "MQTT listening..."
-$BIN_PATH/mosquitto_sub -I $clientID -h $mqtthost $auth -v -t $topic/+/+/set | while read line; do
+$BIN_PATH/mosquitto_sub -I $clientID -h $mqtthost $auth -v -t $topic/+/+/set \
+--will-topic $topic/$online --will-retain --will-qos 1 --will-payload 'false' \
```

Comment:
didn't try yet, but isn't there a backslash missing? `$topic\/$online`
It anyway should be `$state=lost`.